appsmithorg/appsmith
# fix: unexpected return error

## Description

Handle empty new lines in bindings

Fixes #8658

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
Test A:
- unit tests

Test B:
- enter only new line in any js field with {{}} binding 
```
{{

}}
```
- notice "Unexpected return error" doesn't show up anymore

## Checklist:


















## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>


    // Code coverage diff between base branch:release and head branch: fix/unexpected-return-error 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 55.14 **(0)** | 37.09 **(0.02)** | 33.96 **(0)** | 55.64 **(0)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.94 **(0.23)** | 41.67 **(0.84)** | 36.21 **(0)** | 56.99 **(0.25)**
 :green_circle: | app/client/src/workers/evaluate.ts | 88.17 **(0.26)** | 78.13 **(1.46)** | 77.78 **(0)** | 87.5 **(0.29)**</details>
